tinygrad/tensor.py

```diff
@@ -680,6 +680,12 @@ def dropout(self, p=0.5) -> Tensor:
     mask = (Tensor.rand(*self.shape, requires_grad=False) >= p).cast(dtypes.bool)
     return self * mask * (1/(1.0 - p))
 
+  @staticmethod
```

Comment:
Instead of staticmethod, just make the query be `self`

Add type signatures to this function, and I think it's good!